detox/ios/Detox/DetoxInstrumentsManager.m

```diff
@@ -146,7 +149,14 @@ - (id)_configForDetoxRecordingWithURL:(NSURL*)URL
 	[config setRecordingFileURL:URL];
 	
 	//TODO: Finalize the actual config for Detox perf recording.
-	[config setRecordNetwork:NO];
+	[config setRecordEvents:YES];
+	if ([config respondsToSelector:@selector(setRecordInternalReactNativeEvents:)])
```

Comment:
In what cases setRecordInternalReactNativeEvents doesn't exist?

Reply:
When users have older version of Detox Instruments. The supported version was released only yesterday.